ethcore/src/engines/mod.rs

```diff
@@ -71,6 +71,8 @@ pub trait Engine : Sync + Send {
 	/// Block transformation functions, after the transactions.
 	fn on_close_block(&self, _block: &mut ExecutedBlock) {}
 
+	/// If true, generate_seal has to be implemented.
+	fn seals_internally(&self) -> bool { false }
```

Comment:
I haven't noticed it personally, in general it is typical Rust style to provide a line of blank space between each function definition, at least if there are docs. (see an example from `std`: [Iterator](https://doc.rust-lang.org/stable/src/core/up/src/libcore/iter/iterator.rs.html#38-2062) )